platform/marvell-arm64/sonic-platform-modules-supermicro/sse-g3748/scripts/sse-g3748-init.sh

```diff
@@ -0,0 +1,259 @@
+    sudo i2cset -y -r ${NUM_I2C_1_DEVICE} ${I2C_ADDR_IO_PCAL6524} ${PCAL6524_REG_PORT1_DIR} 0xBB
+    sudo i2cset -y -r ${NUM_I2C_1_DEVICE} ${I2C_ADDR_IO_PCAL6524} ${PCAL6524_REG_PORT2_DIR} 0xBB
+    sudo i2cset -y -r ${NUM_I2C_1_DEVICE} ${I2C_ADDR_IO_PCAL6524} ${PCAL6524_REG_PORT0_POL} 0x00
+    sudo i2cset -y -r ${NUM_I2C_1_DEVICE} ${I2C_ADDR_IO_PCAL6524} ${PCAL6524_REG_PORT1_POL} 0x00
+    sudo i2cset -y -r ${NUM_I2C_1_DEVICE} ${I2C_ADDR_IO_PCAL6524} ${PCAL6524_REG_PORT2_POL} 0x00
+
+    echo "Init SFP RS0 and RS1 IO Expander"
+    sudo i2cset -y -r ${NUM_I2C_1_DEVICE} ${I2C_ADDR_IO_PCAL6416} ${PCAL6416_REG_PORT0_DIR} 0x00
+    sudo i2cset -y -r ${NUM_I2C_1_DEVICE} ${I2C_ADDR_IO_PCAL6416} ${PCAL6416_REG_PORT1_DIR} 0x00
+    sudo i2cset -y -r ${NUM_I2C_1_DEVICE} ${I2C_ADDR_IO_PCAL6416} ${PCAL6416_REG_PORT0_POL} 0x00
+    sudo i2cset -y -r ${NUM_I2C_1_DEVICE} ${I2C_ADDR_IO_PCAL6416} ${PCAL6416_REG_PORT1_POL} 0x00
+    sudo i2cset -y -r ${NUM_I2C_1_DEVICE} ${I2C_ADDR_IO_PCAL6416} ${PCAL6416_REG_PORT0_OUT} 0xFF
+    sudo i2cset -y -r ${NUM_I2C_1_DEVICE} ${I2C_ADDR_IO_PCAL6416} ${PCAL6416_REG_PORT1_OUT} 0xFF
+}
+
+# - Main entry
+
+# Install kernel drivers required for i2c bus access
+load_kernel_drivers
+#entropy setting
+#python /etc/entropy.py
+
+# LOGIC to enumerate SFP eeprom devices - send 0x50 to kernel i2c driver - initialize devices
+# the mux may be enumerated at number 4 or 5 so we check for the mux and skip if needed
+echo pca9548 ${I2C_ADDR_MUX_9548_CH0} > ${PATH_I2C_1_DEVICE}/new_device
+
+# Get list of the mux channels
+ismux_bus=$(i2cdetect -l|grep mux|cut -f1)
+
+# Enumerate the SFP eeprom device on each mux channel
+#echo optoe2 0x50 > /sys/bus/i2c/devices/i2c-0/new_device
+for mux in ${ismux_bus}
+do
+    if [[ "${mux}" != "i2c-8" ]] && [[ "${mux}" != "i2c-9" ]]; then
+        echo optoe2 ${I2C_ADDR_QSFP_EEPROM} > /sys/class/i2c-adapter/${mux}/new_device
+	mux_num=$(echo ${mux} | cut -b5)
+	#sleep 1
+	chmod 644 /sys/class/i2c-adapter/${mux}/${mux_num}-0050/eeprom
+    fi
+
+    if [ "${mux}" == "i2c-8" ]; then
+        # Enumerate fan eeprom devices
+        echo eeprom 0x56 > /sys/class/i2c-adapter/i2c-8/new_device
+        #sleep 1
+        chmod 644 /sys/class/i2c-adapter/i2c-8/8-0056/eeprom
+
+        echo eeprom 0x57 > /sys/class/i2c-adapter/i2c-8/new_device
+        #sleep 1
+        chmod 644 /sys/class/i2c-adapter/i2c-8/8-0057/eeprom
+    fi
+
+    if [ "${mux}" == "i2c-9" ]; then
+        # Enumerate fan eeprom devices
+        echo eeprom 0x56 > /sys/class/i2c-adapter/i2c-9/new_device
+        #sleep 1
+        chmod 644 /sys/class/i2c-adapter/i2c-9/9-0056/eeprom
+
+        echo eeprom 0x57 > /sys/class/i2c-adapter/i2c-9/new_device
+        #sleep 1
+        chmod 644 /sys/class/i2c-adapter/i2c-9/9-0057/eeprom
+    fi
+done
+
+# Enumerate rtc 
+echo  pcf8563 0x51 > /sys/bus/i2c/devices/i2c-0/new_device
+# Enumerate system eeprom
+echo 24c02 0x54 > /sys/class/i2c-adapter/i2c-0/new_device
+#sleep 1
```

Comment:
lot of commented code, if not needed anymore clean up

Reply:
removed commented code.